unitary/examples/quantum_rpg/classes_test.py

```diff
@@ -0,0 +1,18 @@
+import unitary.examples.quantum_rpg.classes as classes
```

Comment:
Can we have some tests for the effects of the actions?

Reply:
Done.  Good call.  This caught two issues.